docs/user/troubleshooting.md

```diff
@@ -50,9 +50,9 @@ sudo crictl pods --name=etcd-member
 
 If no pods are shown, etcd will need to be [investigated](#etcd-is-not-running).
 
-### Unable to SSH into Master Nodes
+### Unable to access the Master Nodes via SSH 
```

Comment:
nit: trailing whitespace (and also in the next paragraph).

I'd rather leave this header alone (since I'm fine with "SSH" as a verb).  But if you want to update it, you'll also need to update references to it:

```console
$ git grep unable-to origin/pr/795
origin/pr/795:docs/user/troubleshooting.md:If SSH access to the master nodes isn't available, that will need to be [investigated next](#unable-to-ssh-into-master-node).
```

And it looks like that link is already broken in master because it uses `-node` instead of `-nodes`.  I've filed #980 to fix that, so you don't have to worry about it if you leave the header alone.